specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -61,6 +102,20 @@ model AnalyzeTextOptions {
   @added(ContentSafety.Versions.v2023_10_30_Preview)
   @doc("The incidents to detect.")
   incidents?: IncidentOptions;
+
+  @added(ContentSafety.Versions.v2024_01_30_Preview)
+  @doc("The customized categories will be analyzed. If they are not assigned, the analysis results for all available customized categories will be returned.")
+  customizedCategories?: CustomizedCategory[];
+}
+
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+@doc("Customized category for inference.")
+model CustomizedCategory {
+  @doc("Category name")
+  categoryName: string;
+
+  @doc("Model ID for the category. If not provided, the latest available model will be used.")
+  modelId: string;
```

Comment:
You say, "If not provided", but `modelId` is required. Sure you didn't mean `modelId?: string;`?